Sources/Paywalls/Components/PaywallStackComponent.swift

```diff
@@ -21,23 +21,32 @@ public extension PaywallComponent {
 
         let type: ComponentType
         public let components: [PaywallComponent]
+        public let width: WidthSize?
         public let spacing: CGFloat?
         public let backgroundColor: ColorInfo?
         public let dimension: Dimension
         public let padding: Padding
+        public let margin: Padding
+        public let cornerRadiuses: CornerRadiuses
 
         public init(components: [PaywallComponent],
                     dimension: Dimension = .vertical(.center),
+                    width: WidthSize?,
                     spacing: CGFloat?,
                     backgroundColor: ColorInfo?,
-                    padding: Padding
+                    padding: Padding,
+                    margin: Padding,
+                    cornerRadiuses: CornerRadiuses
```

Comment:
What do you think of giving these default values in the initializer? It makes creating paywall samples via code much quicker/easier, and I think there will be a continued need to do so as we expand the components' properties/capabilities.